DigitalLearningSolutions.Web/Helpers/CookieBannerHelper.cs

```diff
@@ -0,0 +1,46 @@
+﻿using DigitalLearningSolutions.Data.Models.TrackingSystem;
+using DigitalLearningSolutions.Web.ViewModels.Frameworks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using System;
+
+namespace DigitalLearningSolutions.Web.Helpers
+{
+    public static class CookieBannerHelper
+    {
+        public static readonly int CookieExpiryDays = 365;
+        public static readonly string CookieName = "Dls-cookie-consent";

```

Comment:
We probably need to read this from appsettings to allow this code to be moved to a nuget package.

Reply:
Done